test/module/shared_model/backend_proto/proto_block_factory_test.cpp

```diff
@@ -40,7 +40,11 @@ TEST_F(ProtoBlockFactoryTest, UnsafeBlockCreation) {
   std::vector<shared_model::proto::Transaction> txs;
   txs.emplace_back(iroha::protocol::Transaction{});
 
-  auto block = factory->unsafeCreateBlock(height, prev_hash, created_time, txs);
+  std::vector<shared_model::crypto::Hash> rejected_txs{
+      shared_model::crypto::Hash::fromHexString("rubble_devaluation")};
```

Comment:
:) you could actually just create empty vector, but it's fine if you want to keep that